lib/matplotlib/text.py

```diff
@@ -768,7 +794,8 @@ def draw(self, renderer):
             gc.set_url(textobj._url)
             textobj._set_gc_clip(gc)
 
-            angle = textobj.get_rotation()
+            angle = (textobj.get_rotation() + (180.0*transangle/np.pi))%360.0
+
```

Comment:
Sorry, yes, the PR as a whole. I was replying via email.

On Tue, Mar 1, 2016 at 4:35 PM, Elliott Sales de Andrade <
notifications@github.com> wrote:

> In lib/matplotlib/text.py
> <https://github.com/matplotlib/matplotlib/pull/6081#discussion_r54639626>:
>
> > @@ -768,7 +794,8 @@ def draw(self, renderer):
> >              gc.set_url(textobj._url)
> >              textobj._set_gc_clip(gc)
> >
> > -            angle = textobj.get_rotation()
> > +            angle = (textobj.get_rotation() + (180.0*transangle/np.pi))%360.0
> > +
>
> @WeatherGod <https://github.com/WeatherGod> I'm not sure that you wanted
> to comment on this line specifically and not the PR as a whole?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/6081/files#r54639626>.
>